docs/content/docs/2.components/chat.md

```
@@ -38,23 +38,41 @@ Install the required dependencies:
 
 ```bash [bun]
-bun add ai @ai-sdk/gateway @ai-sdk/vue
+bun add ai @ai-sdk/gateway @ai-sdk/vue @comark/nuxt
 ```
 
 ::
 
+Then add `@comark/nuxt` to your modules:
+
+::framework-only
+#nuxt
```

markdownlint-cli2 (0.22.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)